policyengine_us/parameters/gov/states/az/tax/income/credits/family_tax_credits/max_amount.yaml

```diff
@@ -0,0 +1,25 @@
+description: Arizona provides the following maximum amount under the arizona family tax credit based on filing status.
+metadata:
+  period: year
+  unit: currency-USD
+  label: Arizona family tax credit maximum amount
+  breakdown:
+    - filing_status
+SINGLE:
+  2021-01-01: 120
+HEAD_OF_HOUSEHOLD:
+  2021-01-01: 240
+JOINT:
+  2021-01-01: 240
+WIDOW:
+  2021-01-01: 240
+SEPARATE:
+  2021-01-01: 120
+reference:
+  - title: 43-1073. Family income tax credit (B)
+    href: https://www.azleg.gov/ars/43/01073.htm
+  - title: 2022 Arizona Tax Form Instructions Line50
+    href: https://azdor.gov/sites/default/files/2023-03/FORMS_INDIVIDUAL_2022_140i-2D.pdf#page22
```

Comment:
```suggestion
    href: https://azdor.gov/sites/default/files/2023-03/FORMS_INDIVIDUAL_2022_140i-2D.pdf#page23
```

Reply:
please check page numbers in pdf files, the value of the specific parameter has to appear on that specific page